srcpkgs/66/patches/man_page_fix.diff

```diff
@@ -0,0 +1,40 @@
+diff --git a/doc/make-man.sh b/doc/make-man.sh
```

Comment:
Would be nice to get a new release and avoid so much patching :/

Reply:
I agree, I have send my patch upstream :)

Reply:
Patch accepted: https://framagit.org/Obarun/66/-/commit/c0af2428c8271646b9fd4927f860b7ff6a28e3db